seed/static/seed/js/controllers/menu_controller.js

```diff
@@ -53,11 +53,9 @@ angular.module('BE.seed.controller.menu', [])
       };
       $scope.$on('app_error', function (event, data) {
         // Keep the first error
-        if (!$rootScope.route_load_error) {
-          $rootScope.route_load_error = true;
-          $scope.menu.error_message = data.message;
-          $rootScope.load_error_message = data.data.message;
-        }
+        if (!$rootScope.route_load_error) $rootScope.route_load_error = true;
+        $scope.menu.error_message = data.message;
+        $rootScope.load_error_message = data.data.message;
```

Comment:
Looks like inadvertent formatting - could you change it back? Probably worth you reviewing the FE code for things like this as I might've missed them.

Reply:
This was intentional. Or error message doesn't get updated with respect to different errors. As a matter of fact this is a mini bug that needs to be improved in case multiple error on page found.

Reply:
<img width="1219" alt="Screen Shot 2020-05-07 at 7 02 52 PM" src="https://user-images.githubusercontent.com/49968203/81359058-57747c00-9095-11ea-90e5-7b05eb9fe283.png">